.github/workflows/branch-name.yaml

```
@@ -12,16 +16,19 @@ jobs:
     steps:
       - name: Validate branch name format
         shell: bash
+        env:
+         GH_TOKEN: ${{ github.token }}
         run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)